### PR TITLE
component TodoList added and Route added in App.js

### PR DESCRIPTION
**Pull request by R1SsophieFEHU**
component TodoList added and Route added in App.js

### CODE REVIEW

File: front/src/components/TodoList/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            />
+          <button className="btnAdd" type="submit">Add</button>
+        </form>
+        <TodoList todos={this.state.todos} removeItem={this.removeItem} />
+      </div>
+    );
+  }
+}
+
+class TodoList extends Component {
```
Comment (ghost) [inline front/src/components/TodoList/TodoList.jsx:5]: Put this component in a separate file (so each file contains one single component)

File: front/src/components/TodoList/TodoList.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  render() {
+    return(
+      <div className="list">{ this.props.todos.map((todo, index) => {
+          return <div className="items">
+                  { todo }
+                  <button className="btnDelete" onClick={(e) => { this.props.removeItem(index)}} key={todo}>Del</button>
```
Comment (ghost) [inline front/src/components/TodoList/TodoList.jsx]: You cannot guarantee that todo will be unique, so it's dangerous to set it as key

**Review by ghost — CHANGES_REQUESTED** [review]

File: front/src/components/TodoList/TodoList.jsx (hunk 2 above)
Comment (ghost) [reply front/src/components/TodoList/TodoList.jsx]: It's ok now, just remove this key prop which now isn't necessary anymore

**Review by ghost — CHANGES_REQUESTED** [review]

File: front/src/components/TodoList/TodoList.jsx (hunk 2 above)
Comment (R1SsophieFEHU) [reply front/src/components/TodoList/TodoList.jsx]: Ok, done.

**Pull request merged**